test/images/agnhost/netexec/netexec.go

```diff
@@ -338,7 +338,9 @@ func exitHandler(w http.ResponseWriter, r *http.Request, exitCh chan<- shutdownR
 
 func hostnameHandler(w http.ResponseWriter, r *http.Request) {
 	log.Printf("GET /hostname")
-	fmt.Fprint(w, getHostName())
+	hostname := getHostName()
+	log.Printf("response: %s", hostname)
```

Comment:
I think we can add the log.Printf inside the `getHostName` function so it will get always printed independently from where it is called, UDP server also uses it per example